src/main/java/uk/gov/hmcts/reform/prl/models/dto/ccd/ManageOrders.java

```diff
@@ -278,4 +278,7 @@ public class ManageOrders {
     //PRL-3254 - Added for populating hearing dropdown
     private DynamicList hearingsType;
 
+    @JsonProperty("isFinalOrderIssuedForAllChildren")
+    private YesOrNo isFinalOrderIssuedForAllChildren;
```

Comment:
Why do we need JsonProperty and why do we need setter? setter was not required earlier?